aws/data_source_aws_secretsmanager_secret_version.go

```diff
@@ -76,7 +77,11 @@ func dataSourceAwsSecretsManagerSecretVersionRead(d *schema.ResourceData, meta i
 
 	d.SetId(fmt.Sprintf("%s|%s", secretID, version))
 	d.Set("secret_id", secretID)
-	d.Set("secret_string", output.SecretString)
+	var sec map[string]interface{}
+	if err = json.Unmarshal([]byte(*output.SecretString), &sec); err != nil {
+		return fmt.Errorf("Error getting secret string: %s", err)
```

Comment:
It is valid to create a Secrets Manager secret that is a single string value (not JSON). This error should only be logged to not break handling with those types of secrets.